src/ray/common/scheduling/scheduling_ids.cc

```diff
@@ -30,7 +30,7 @@ int64_t StringIdMap::insert(const string sid, bool test) {
   }
 };
 
-void StringIdMap::remove(const string sid) {
+void StringIdMap::remove(const string& sid) {
```

Comment:
Please update these function names to match style guide as well (maybe in a separate PR)

Reply:
Done